crates/build/re_types_builder/src/codegen/python/mod.rs

```diff
@@ -2649,8 +2649,8 @@ fn quote_columnar_methods(reporter: &Reporter, obj: &Object, objects: &Objects)
             for batch in batches:
                 arrow_array = batch.as_arrow_array()
 
-                # For primitive arrays, we infer partition size from the input shape.
-                if pa.types.is_primitive(arrow_array.type):
+                # For primitive arrays and fixed size list arrays, we infer partition size from the input shape.
+                if pa.types.is_primitive(arrow_array.type) or pa.types.is_fixed_size_list(arrow_array.type):
```

Comment:
it's technically fixed sized lists of primitive arrays, no? 🤔

Reply:
yes. good catch would come back to haunt us I think.

Reply:
we concluded on a Zoom call that this actually doesn't matter at all here - if we had any fixed sized non primitive arrays it should just work, but alas we don't have any to begin with